src/triage/experiments/multicore.py

```diff
@@ -27,14 +41,23 @@ def __init__(self, config, db_engine, *args, n_processes=1, n_db_processes=1, **
             raise ValueError("n_processes must be 1 or greater")
         if n_db_processes < 1:
             raise ValueError("n_db_processes must be 1 or greater")
-        if n_db_processes == 1 and n_processes == 1:
+        if n_bigtrain_processes < 1:
+            raise ValueError("n_bigtrain_processes must be 1 or greater")
+        if n_db_processes == 1 and n_processes == 1 and n_bigtrain_processes == 1:
             logger.notice(
                 "Both n_processes and n_db_processes were set to 1. "
                 "If you only wish to use one process to run the experiment, "
                 "consider using the SingleThreadedExperiment class instead"
             )
         self.n_processes = n_processes
         self.n_db_processes = n_db_processes
+        self.n_bigtrain_processes = n_bigtrain_processes
+        self.n_processes_lookup = {
+            BatchKey.QUICKTRAIN: self.n_db_processes,
+            BatchKey.BIGTRAIN: self.n_bigtrain_processes,
+            BatchKey.MAYBETRAIN: self.n_db_processes
```

Comment:
Shouldn't this be `self.n_processes` not `self.n_db_processes` for `QUICKTRAIN` and `MAYBETRAIN`?

Reply:
Yes, you're right